src/Serialization/containers.jl

```diff
@@ -533,3 +533,31 @@ function load_object(s::DeserializerState, ::Type{<: Set}, params::Ring)
   end
   return Set{T}(loaded_entries)
 end
+
+################################################################################
+# Sparse rows
+
+@register_serialization_type SRow uses_params
+
+function save_object(s::SerializerState, obj::SRow)
+  save_data_array(s) do
+    for (i, v) in collect(obj)
+      save_object(s, (i, v))
+    end
+  end
+end
+
+function load_object(s::DeserializerState, ::Type{<:SRow}, params::Ring)
+  pos = Int[]
+  entry_type = elem_type(params)
+  values = entry_type[]
+  load_array_node(s) do _
+    push!(pos, load_object(s, Int, 1))
+    if serialize_with_params(entry_type)
+      push!(values, load_object(s, entry_type, params, 2))
+    else
+      push!(values, load_object(s, entry_type, 2))
+    end
+  end
+  return sparse_row(params, pos, values)
```

Comment:
```suggestion
  return sparse_row(params, pos, values; sort=false)
```
Since we assume that the file already has the correct format, we can assume here that the positions in `pos` are already sorted, and so Hecke does not need to do that again

Reply:
And of course the same in https://github.com/oscar-system/Oscar.jl/blob/251b831144612cb08fcde2be58d40f59d6a15fac/src/Serialization/Rings.jl#L438 and https://github.com/oscar-system/Oscar.jl/blob/251b831144612cb08fcde2be58d40f59d6a15fac/src/Serialization/Rings.jl#L446.